config/initializers/lograge.rb

```diff
@@ -3,13 +3,7 @@
 if %w[development test].include? Rails.env
   Rails.application.configure do
     config.lograge.enabled = true
-    config.lograge.custom_options = lambda do |event|
```

Comment:
Reset this file and open a new ticket for removing LogRage. In it ping Ryan O about if we're using any existing LR functionality